Task 2/Task 2.1/Task 2.1.2/Figures/RoundShapes/Circle.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Task_2._1._2.Figures.RoundShapes
+{
+    public class Circle : Figure
+    {
+        public Point Centre { get; }
+        public int Radius { get; }
+        public Circle(Point p, int r) : base("Окружность")
+        {
+            Centre = p;
+            Radius = r;
+        }
```

Comment:
As a developer I can create a Circle without using your helpers - in this case I can make a mistake and create a circle with negative radius.
Incapsulation - class is the closest and most needed line of defense against incorrect object state. Here it should've check radius too.